src/AspNetCore/Src/forwardinghandler.cxx

```diff
@@ -1151,9 +1152,27 @@ FORWARDING_HANDLER::OnExecuteRequestHandler(
         hr = ((IN_PROCESS_APPLICATION*)m_pApplication)->LoadManagedApplication();
         if (FAILED(hr))
         {
+            _com_error err(hr);
+            if (ANCMEvents::ANCM_START_APPLICATION_FAIL::IsEnabled(m_pW3Context->GetTraceContext()))
```

Comment:
By removing IsEnabled, this would turn into:
```
 ANCMEvents::ANCM_START_APPLICATION_FAIL::RaiseEvent(
                    m_pW3Context->GetTraceContext(),
                     NULL,
                     err.ErrorMessage());
```
and would noop if it isn't enabled.

Reply:
As I said earlier, the FREB header file was generated automatically by a tool. The using pattern is old. We have to call IsEnable first and then write the trace.

Reply:
Oh ewwwwwwwwwwwwwwww.

Reply:
Ahh I'll dismiss my reviews for it then.